Makefile

```diff
@@ -1,11 +1,129 @@
-UNITS = "@unit.abijson or @unit.abijson.byname or @unit.algod or @unit.algod.ledger_refactoring or @unit.applications or @unit.atc_method_args or @unit.atomic_transaction_composer or @unit.dryrun or @unit.dryrun.trace.application or @unit.feetest or @unit.indexer or @unit.indexer.ledger_refactoring or @unit.indexer.logs or @unit.offline or @unit.rekey or @unit.transactions.keyreg or @unit.responses or @unit.responses.231 or @unit.tealsign or @unit.transactions or @unit.transactions.payment or @unit.responses.unlimited_assets or @unit.sourcemap"
+UNITS = "@unit.abijson or @unit.abijson.byname or @unit.algod or @unit.algod.ledger_refactoring or @unit.applications or @unit.atc_method_args or @unit.atomic_transaction_composer or @unit.dryrun or @unit.dryrun.trace.application or @unit.feetest or @unit.indexer or @unit.indexer.ledger_refactoring or @unit.indexer.logs or @unit.offline or @unit.rekey or @unit.transactions.keyreg or @unit.responses or @unit.responses.231 or @unit.responses.unlimited_assets or @unit.sourcemap or @unit.tealsign or @unit.transactions or @unit.transactions.payment "
 unit:
 	behave --tags=$(UNITS) tests -f progress2
 
-INTEGRATIONS = "@abi or @algod or @applications or @applications.verified or @assets or @auction or @c2c or @compile or @dryrun or @dryrun.testing or @indexer or @indexer.231 or @indexer.applications or @kmd or @rekey_v1 or @send.keyregtxn or @send or @compile.sourcemap"
+INTEGRATIONS = "@abi or @algod or @applications or @applications.verified or @assets or @auction or @c2c or @compile or @compile.sourcemap or @dryrun or @dryrun.testing or @kmd or @rekey_v1 or @send or @send.keyregtxn"
 integration:
-	behave --tags=$(INTEGRATIONS) tests -f progress2
+	behave --tags=$(INTEGRATIONS) tests -f progress2 --no-capture
+
+harness:
+	./test-harness.sh
 
 PYTHON_VERSION ?= 3.8
-docker-test:
-	PYTHON_VERSION='$(PYTHON_VERSION)' ./run_integration.sh
+docker-pysdk-build:
+	docker build -t py-sdk-testing --build-arg PYTHON_VERSION="${PYTHON_VERSION}" .
+
+docker-pysdk-run:
+	docker ps -a
+	docker run -it --network host py-sdk-testing:latest
+
+docker-test: harness docker-pysdk-build docker-pysdk-run
+
+# algorand-sdk-testing's git branch:
+TBRANCH = master
+# go-algorand's git branch:
+ABRANCH = master
+get-trepo:
+	rm -rf test-harness && rm -rf tests/features \
+	&& git clone --single-branch --branch $(TBRANCH) https://github.com/algorand/algorand-sdk-testing.git test-harness \
+	&& sed -i "" -e "s#ALGOD_BRANCH=\".*\"#ALGOD_BRANCH=\"$(ABRANCH)\"#g" test-harness/.up-env \
+	&& mkdir -p test/features \
+	&& cp -r test-harness/features/* test/features
+
+up:
+	./test-harness/scripts/up.sh
+
+down:
+	./test-harness/scripts/down.sh
+
+restart-test-harness: get-trepo up
+
+ps:
+	cd test-harness \
+	&& docker-compose ps
+
+pause:
+	cd test-harness \
+	&& docker-compose pause 
+
+unpause:
+	cd test-harness \
+	&& docker-compose unpause 
+
+
+# sdk-harness-algod \
```

Comment:
Is everything below this necessary? If we decide to include it, it might be nice to have a comment to explain what they do

Reply:
OOPS.... didn't mean to stage/push these changes. These are some shortcuts I use locally. Will revert shortly.